src/args.ts

```diff
@@ -53,7 +58,7 @@ export const getYargs = async (): Promise<ReviewArgs> => {
     .option("model", {
       description: "The model to use for generating the review.",
       type: "string",
-      default: "gpt-4",
+      default: "gpt-3.5-turbo",
```

Comment:
[Note] I would argue in favour of having this as the default until we have access to gpt-4 as standard, as with gpt-4 as our default the script on main and tests are not working

Reply:
I am still against this as it will not encourage other people using the app to use gpt4. 

When we publish this we will be changing functionality for the people using the project via npm. Results are much worse with gpt-3.5.

Reply:
I mean I will change it but this issue may be worth flagging in documentation

Reply:
Changed! :)

Reply:
Defaulting to gpt-4 is not breaking our tests. We can specify the model with the --model flag. Our app is not well optimised for gpt-3.5. That is worth flagging in a ticket definitely.